### DISCUSSION TITLE
[Bug] Ordered list with items more than 10 - error nested lists

### DISCUSSION THREAD

**Discussion by HeKr09** [dotnet/docfx#9368] [General]
**Describe the bug**
I have got an ordered list with eleven list items (li). Within the 10. item I have got a nested list with three items. Those three items are not generated as separate list but within the parent list.

**To Reproduce**
Markdown like this:
```markdown
## Something
Content
1.  One
    1.  One nested
    2.  Two nested
2.  Two
    1.  One nested
    2.  Two nested
    3.  Three nested
    4.  Four nested
3.  Three
4.  Four
    1.  One Nested
5.  Five
6.  Six
    1.  One nested
    2.  Two nested
    3.  Three nested
7.  Seven
8.  Eight
9.  Nine
10.  Ten
    1.  One Nested
    2.  Two Nested
    3.  Three Nested
11.  Eleven
```

**Expected behavior**
A html output like:
```html
<ol>
   <li>One</li>
   ...
   <li>Ten</li>
   <ol>
      <li>One Nested</li>
      <li>Two Nested</li>
      <li>Three Nested</li>
   </ol>
   <li>Eleven</li>
</ol>
```

**Context (please complete the following information):**
- OS: Windows 11
- Docfx version: 2.70.0

**Additional Information**
As I can see: The markdown above is also not showing correct, isn't it?

**Comment by yufeih** [comment]
Delete the extra space before "Ten" should work. It is markdown problem.

**Comment by HeKr09** [comment]
OMG... Thank you very much @yufeih !!! I wonder what would happen at 100.... ;)
